meetings/2021-09-08.md

```diff
@@ -0,0 +1,140 @@
+We will evaluate it on how it gives restrictions on clients.
+
+**Henry** Note with HttpSig the Wallet can have policies about which apps it gives signtatures to for what sets of resources, so that access control does not need to be done in server ACRs.
+
+There seems to be a requirement for Solid-OIDC that servers need to be authorizing clients.
+This is not needed for HTTPSig. 
+I am ok to support the Solid OIDC use case.
+
+**Aaron** I see authorizing user vs clients as different dimensions. One would be authorizing users
+/ social agents. The other one is authorizing clients. As soon as authorization system can
+identify both it can do its job. I know that ACP allows that.
+
+**Henry**: A so if we are speaking of setting access control rules that the Wallet agent can use before signing HTTP headers, I think we should be able to use the WAC ontology to do that. The Wallet needs to know which set of resoures a client can access in which modes before signing the headers. 
+
+**Pavlik**  Ah perhaps you can write that up as an issue Henry.
+
+I would like to propose using GNAP terminology of End-user, client (instance), Resource Owner.
+
+
+### [Access Modes](https://github.com/solid/authorization-panel/issues/253)
+
+**Justin** https://solid.github.io/authorization-panel/authorization-ucr/#collection-readcreatedelete
+
+**Justin** We have modes which are overloaded. Given write one can delete, create, update.
+This leads to over permissioning agents. I think this should be more urgent issue to resolve.
+I think aligning it around CRUD is not a radical proposition. Kjetilk has done really good job
+describing the issue. I recall Matthieu suggesting to put it in separate document.
+
+**Pavlik** I'm not sure if protocol spec has to define both operations and access modes. Also how do we imagine mapping between those two?
+
+**Henry** 
+ It is a great discussion. But it is branching a lot. Whould we move it over to the discussions area, which also has threading?
+
+ I am trying to find mathematical backing for the decisions we could make, so that we don't have to jump back and forward all the time. I think we have clearly Read/Write (those come from Lenses) and there could be subsets under Write for Create Update Delete. 
+Delete is an interesting question: does it need permissions on the container?
+
+**Justin** I don't forsee coming back 6 months later or 6 years later and saying - i need another mode.
+There has been some arguments that we will add some later, I think that our current set needs improvement
+but after that they should possibly last forever.
+
+**Barath** I agree, that well chosen modes will last for long time. Even substandard modes serve in existing systems.
+
+**Justin** I think delete is good example, it can be set on the level of container and on individual resources. I don't see argument that there could be delete without implicit update of containment triples.
```

Comment:
```suggestion
**Justin** I think `delete` is good example; it can be set on the level of container and on individual resources. I don't see argument that there could be `delete` without implicit update of containment triples.
```